src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).orElseThrow();
```

Comment:
Add exception you are throwing here or use just get() then  NoSuchElementException will be thrown

Reply:
orElseThrow() was added as a more transparent synonym for get(), it has the same functionality.